src/Poseidon/CLI/Checksumupdate.hs

```diff
@@ -0,0 +1,26 @@
+module Poseidon.CLI.Checksumupdate (
```

Comment:
What's the difference between this new CLI module and the former `update`?

Reply:
Oh snap - that was accidentally copied from #70 :facepalm:. Maybe you could also give this other PR your blessing. Then I can merge them both at once. `checksumupdate` is just a clearer name for the former `update`.